contracts/contracts/utilities/DomainObjs.sol

```diff
@@ -8,6 +8,12 @@ contract DomainObjs {
   /// @notice the length of a MACI message
   uint8 public constant MESSAGE_DATA_LENGTH = 10;
 
+  /// @notice voting modes
+  enum Mode {
```

Comment:
just a stupid question: why we call it EMode elsewhere, but use Mode here?

Reply:
Because it's solidity, not js/ts